LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/inline-child-with-composited-filter.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html class=reftest-wait>
+<title>View transitions: inline child with filter</title>
+<link rel="help" href="https://www.w3.org/TR/css-view-transitions-1/">
+<link rel="author" href="mailto:vmpstr@chromium.org">
```

Comment:
Is this a new test or we just importing it? Do we need to add it to w3c-import.log?

Reply:
It's a new test, but I left the original author there since it's mostly a duplicate of an existing test (with the addition of forcing compositing the filtered element).